js/pages/itemdetails.js

```diff
@@ -436,20 +436,87 @@ pageInit.itemdetails = function(category, assetId) {
 			})
 		}
 	}
-	
+
+	if(settings.itemdetails.showRevenue) {
+		const elem = html`
+		<div class="clearfix item-field-container">
+			<div class="text-label text-overflow field-label">Revenue</div>
+			<span class="icon-robux-16x16"> </span> <span class="field-content-revenue"></div>
```

Comment:
Keeping the field-content tag is quite important as Roblox has some CSS attached to it. 
```suggestion
			<span class=field-content><span class="icon-robux-16x16"></span> <span class="field-content-revenue"></span></span>
```